django/db/backends/sqlite3/creation.py

```diff
@@ -4,11 +4,15 @@
 
 from django.core.exceptions import ImproperlyConfigured
 from django.db.backends.base.creation import BaseDatabaseCreation
+from django.utils.encoding import force_text
 from django.utils.six.moves import input
 
 
 class DatabaseCreation(BaseDatabaseCreation):
 
+    def is_database_in_memory(self, database_name):
```

Comment:
> Probably utility methods like this could go toward the bottom of the class rather than the top?

My guideline has usually been to put methods after the methods they depend on so you can read it easier from top to bottom. But I can change that.